src/System.ServiceModel.Federation/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
+    <Target>M:System.ServiceModel.Federation.WSTrustTokenParameters.get_KeySize</Target>
+    <Left>lib/net8.0/System.ServiceModel.Federation.dll</Left>
```

Comment:
I think this is another bug in API compat tooling. WSTrustTokenParameters has the new keyword because when the class was first introduced, the base class didn't have a KeySize property. The base class did have a KeySize property exposed on NetFx and was brought back, so the definition in WSTrustTokenParameters needed the new keyword so that it didn't become binary incompatible. This hasn't changed single 6.1.0.

Reply:
Yeah, it does seem to be off:
```
CP0002: Member 'System.Nullable<int> System.ServiceModel.Federation.WSTrustTokenParameters.KeySize.get' exists on [Baseline] lib/net8.0/System.ServiceModel.Federation.dll but not on lib/netstandard2.0/System.ServiceModel.Federation.dll
```

This has to do with comparing the previous `net8.0` asset with the new `netstandard2.0` asset.  You'll only face this issue across this one baseline comparison.  Details here https://github.com/dotnet/sdk/issues/46834.